README.md

```diff
@@ -154,6 +156,15 @@ sudo cp target/release/alacritty /usr/local/bin # or anywhere else in $PATH
 cp Alacritty.desktop ~/.local/share/applications
 ```
 
+#### OS X
+
+To install the desktop entry for Alacritty, run
+
+```sh
+make app
```

Comment:
That's right, and I'd rather not put the `cargo build --release` into the Makefile since people may want to enable different features (once we have them *cough* scrollback). I think the thing to do here is just add it to the instructions.